### ISSUE TITLE
Relationship field that is labeled 'status' will not save.

### ISSUE THREAD

**Issue by raoulunger** [pods-framework/pods#5440]
**Describe the bug**
When a relationship field is labeled 'status', the field will disappear.

**To Reproduce**
1. Edit a pod
2. Add relationship field
3. label it 'status'
4. Save the field (not the pod!). The field will save as expected, and can be moved around to change the order of fields (provided there are already more fields in the pod.)
5. But: when the entire pod is saved, the relationship field will have disappeared. It is also not present when adding a new pod item, or when adding magical tags in a pod template.

**Expected behavior**
Relationship fields with any other label than 'status' do not show this problem. So it is a specific problem with the word 'status'. It is not in the list of WPs 'Reserved Key Words', so we should be able to use it. It is also a word that is fairly likely to be used in a taxonomy context, so it would be good if this bug were fixed. I took me a day to find out it was a semantic issue.

**Screenshots**
n.a.

## Pods Version
Version 2.7.14

### wp-core ###

version: 5.2.2
site_language: en_US
user_language: en_US
permalink: undefined
https_status: true
user_registration: 0
default_comment_status: closed
multisite: false
user_count: 1
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /var/www/vhosts/ulrikkereedfindenegg.nl/httpdocs
wordpress_size: 96.75 MB (101454912 bytes)
uploads_path: /var/www/vhosts/ulrikkereedfindenegg.nl/httpdocs/wp-content/uploads
uploads_size: 1.35 MB (1420098 bytes)
themes_path: /var/www/vhosts/ulrikkereedfindenegg.nl/httpdocs/wp-content/themes
themes_size: 8.41 MB (8817125 bytes)
plugins_path: /var/www/vhosts/ulrikkereedfindenegg.nl/httpdocs/wp-content/plugins
plugins_size: 17.24 MB (18075106 bytes)
database_size: 4.98 MB (5226496 bytes)
total_size: 128.74 MB (134993737 bytes)

### wp-active-theme ###

name: WP Bootstrap Starter Child
version: 3.2.3.1565018185
author: Afterimage Designs
author_website: https://afterimagedesigns.com/
parent_theme: WP Bootstrap Starter
theme_features: automatic-feed-links, title-tag, post-thumbnails, menus, html5, custom-background, customize-selective-refresh-widgets, custom-header, infinite-scroll, jetpack-responsive-videos, woocommerce, wc-product-gallery-zoom, wc-product-gallery-lightbox, wc-product-gallery-slider, widgets, editor-style
theme_path: /var/www/vhosts/ulrikkereedfindenegg.nl/httpdocs/wp-content/themes/wp-bootstrap-starter

### wp-themes (1) ###

WP Bootstrap Starter: version: 3.2.3, author: Afterimage Designs

### wp-plugins-active (14) ###

Admin Columns: version: 3.4.1, author: AdminColumns.com
All In One WP Security: version: 4.3.9.4, author: Tips and Tricks HQ, Peter Petreski, Ruhul, Ivy
Catch IDs: version: 1.8, author: Catch Plugins
Child Theme Configurator: version: 2.5.0, author: Lilaea Media
Classic Editor: version: 1.5, author: WordPress Contributors
Coming Soon Page & Maintenance Mode by SeedProd: version: 5.0.26, author: SeedProd
Customize Pane Resizer: version: 0.1.1, author: WordPress.org
Duplicate Post: version: 3.2.3, author: Enrico Battocchi
Editor Full Width Gutenberg: version: 1.0.2, author: Anibal Ardid
Insert Headers and Footers: version: 1.4.4, author: WPBeginner
PHP Code Widget: version: 2.3, author: Otto
Pods - Custom Content Types and Fields: version: 2.7.14, author: Pods Framework Team
Really Simple SSL: version: 3.2.3, author: Rogier Lankhorst, Mark Wolters
Widget Shortcode: version: 0.3.4, author: Hassan Derakhshandeh

### wp-media ###

image_editor: WP_Image_Editor_Imagick
imagick_module_version: 1687
imagemagick_version: ImageMagick 6.9.7-4 Q16 x86_64 20170114 http://www.imagemagick.org
imagick_limits: 
	imagick::RESOURCETYPE_AREA: 122 MB
	imagick::RESOURCETYPE_DISK: 1073741824
	imagick::RESOURCETYPE_FILE: 6144
	imagick::RESOURCETYPE_MAP: 512 MB
	imagick::RESOURCETYPE_MEMORY: 256 MB
	imagick::RESOURCETYPE_THREAD: 4
gd_version: bundled (2.1.0 compatible)
ghostscript_version: 9.26

### wp-server ###

server_architecture: Linux 4.9.0-9-amd64 x86_64
httpd_software: Apache
php_version: 7.2.20 64bit
php_sapi: fpm-fcgi
max_input_variables: 1000
time_limit: 30
memory_limit: 256M
max_input_time: 60
upload_max_size: 128M
php_post_max_size: 8M
curl_version: 7.52.1 OpenSSL/1.0.2s
suhosin: false
imagick_availability: true
htaccess_extra_rules: true

### wp-database ###

extension: mysqli
server_version: 5.5.5-10.1.38-MariaDB-0+deb9u1
client_version: mysqlnd 5.0.12-dev - 20150407 - $Id: 3591daad22de08524295e1bd073aceeff11e6579 $

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /var/www/vhosts/ulrikkereedfindenegg.nl/httpdocs/wp-content
WP_PLUGIN_DIR: /var/www/vhosts/ulrikkereedfindenegg.nl/httpdocs/wp-content/plugins
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

### pods ###

pods-server-software: Apache
pods-user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:68.0) Gecko/20100101 Firefox/68.0
pods-session-save-path: /var/lib/php/sessions
pods-session-save-path-exists: No
pods-session-save-path-writable: No
pods-session-max-lifetime: 1440
pods-opcode-cache-apc: No
pods-opcode-cache-memcached: No
pods-opcode-cache-opcache: No
pods-opcode-cache-redis: No
pods-object-cache-apc: No
pods-object-cache-apcu: No
pods-object-cache-memcache: No
pods-object-cache-memcached: No
pods-object-cache-redis: Yes
pods-memory-current-usage: 7.074M
pods-memory-current-usage-real: 6.000M
pods-network-wide: No
pods-install-location: /var/www/vhosts/ulrikkereedfindenegg.nl/httpdocs/wp-content/plugins/pods/
pods-developer: No
pods-tableless-mode: No
pods-light-mode: No
pods-strict: No
pods-allow-deprecated: Yes
pods-api-cache: Yes

`

</details>

## Pods Package Export (helpful!)
{"meta":{"version":"2.7.14","build":1565881313},"pods":{"311":{"id":311,"name":"concert","label":"Concerten","description":"","type":"post_type","storage":"meta","object":"","alias":"","fields":{"huppie":{"id":408,"name":"huppie","label":"Huppie","description":"","help":"","class":"","type":"pick","weight":0,"pick_object":"custom-simple","pick_val":"","sister_id":"","pick_custom":"toekomst\nverleden","required":"1","pick_format_type":"single","pick_format_single":"dropdown","pick_format_multi":"checkbox","pick_allow_add_new":"1","pick_taggable":"0","pick_show_icon":"1","pick_show_edit_link":"1","pick_show_view_link":"1","pick_limit":"0","pick_user_role":[],"pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","pick_select_text":"","pick_table_id":"","pick_table_index":"","pick_display":"","pick_where":"","pick_orderby":"","pick_groupby":""},"titel":{"id":312,"name":"titel","label":"Titel","description":"","help":"","class":"","type":"paragraph","weight":1,"pick_object":"","pick_val":"","sister_id":"","required":"1","paragraph_allow_html":"1","paragraph_oembed":"0","paragraph_wptexturize":"1","paragraph_convert_chars":"1","paragraph_wpautop":"1","paragraph_allow_shortcode":"0","paragraph_allowed_html_tags":"strong em a ul ol li b i","paragraph_max_length":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","paragraph_repeatable":"0","paragraph_placeholder":""},"koor":{"id":313,"name":"koor","label":"Koor","description":"","help":"","class":"","type":"paragraph","weight":2,"pick_object":"","pick_val":"","sister_id":"","required":"1","paragraph_allow_html":"1","paragraph_oembed":"0","paragraph_wptexturize":"1","paragraph_convert_chars":"1","paragraph_wpautop":"1","paragraph_allow_shortcode":"0","paragraph_allowed_html_tags":"strong em a ul ol li b i","paragraph_max_length":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","paragraph_repeatable":"0","paragraph_placeholder":""},"stad":{"id":314,"name":"stad","label":"Stad","description":"","help":"","class":"","type":"paragraph","weight":3,"pick_object":"","pick_val":"","sister_id":"","required":"1","paragraph_allow_html":"1","paragraph_oembed":"0","paragraph_wptexturize":"1","paragraph_convert_chars":"1","paragraph_wpautop":"1","paragraph_allow_shortcode":"0","paragraph_allowed_html_tags":"strong em a ul ol li b i","paragraph_max_length":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","paragraph_repeatable":"0","paragraph_placeholder":""},"locatie":{"id":315,"name":"locatie","label":"Locatie","description":"","help":"","class":"","type":"paragraph","weight":4,"pick_object":"","pick_val":"","sister_id":"","required":"1","paragraph_allow_html":"1","paragraph_oembed":"0","paragraph_wptexturize":"1","paragraph_convert_chars":"1","paragraph_wpautop":"1","paragraph_allow_shortcode":"0","paragraph_allowed_html_tags":"strong em a ul ol li b i","paragraph_max_length":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","paragraph_repeatable":"0","paragraph_placeholder":""},"tijd":{"id":316,"name":"tijd","label":"tijd","description":"","help":"","class":"","type":"time","weight":5,"pick_object":"","pick_val":"","sister_id":"","required":"1","time_type":"12","time_format":"h_mma","time_format_24":"hh_mm","time_allow_empty":"1","time_html5":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","time_repeatable":"0","time_format_custom":"","time_format_custom_js":""},"datum":{"id":317,"name":"datum","label":"Datum","description":"","help":"","class":"","type":"date","weight":6,"pick_object":"","pick_val":"","sister_id":"","required":"1","date_type":"format","date_format":"mdy","date_allow_empty":"1","date_html5":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","date_repeatable":"0","date_format_custom":"","date_format_custom_js":""}},"show_in_menu":"1","label_singular":"Concert","public":"1","show_ui":"1","supports_title":"1","supports_editor":"0","publicly_queryable":"1","exclude_from_search":"0","capability_type":"post","capability_type_custom":"concert","capability_type_extra":"1","has_archive":"0","hierarchical":"0","rewrite":"1","rewrite_with_front":"1","rewrite_feeds":"0","rewrite_pages":"1","query_var":"1","can_export":"1","default_status":"draft","supports_author":"0","supports_thumbnail":"0","supports_excerpt":"0","supports_trackbacks":"0","supports_custom_fields":"0","supports_comments":"0","supports_revisions":"0","supports_page_attributes":"0","supports_post_formats":"0","built_in_taxonomies_category":"1","built_in_taxonomies_link_category":"0","built_in_taxonomies_post_tag":"0","show_in_nav_menus":"1","show_in_admin_bar":"1","pfat_enable":"0","pfat_run_outside_loop":"0","pfat_append_single":"append","pfat_filter_single":"the_content","pfat_append_archive":"append","pfat_filter_archive":"the_content","rest_enable":"1","read_all":"0","write_all":"0","pfat_single":"agenda"}},"templates":{"agenda":{"id":328,"name":"agenda","code":"<div class='{@category}'>\r\n\r\n<h2>{@titel}<\/h2>\r\n<p>{@koor}<\/p>\r\n<p>{@stad}<\/p>\r\n<p>{@locatie}<\/p>\r\n<p>{@tijd}<\/p>\r\n<p>{@datum}<\/p>\r\n\r\n<\/div>\r\n","type":"template","slug":"agenda","options":{"_edit_last":"3","_edit_lock":"1565876127:3","admin_only":"","restrict_capability":"","capability_allowed":"","pod_reference":"a:1:{s:3:\"pod\";s:7:\"concert\";}"}},"agenda_home":{"id":362,"name":"agenda_home","code":"<div class='agenda_home'>\r\n\r\n<h2>{@titel}<\/h2>\r\n<p>{@koor}<\/p>\r\n<p><em>{@stad}<\/em><\/p>\r\n<p>{@locatie}<\/p>\r\n<p>{@tijd}<\/p>\r\n<p>{@datum}<\/p>\r\n\r\n<\/div>\r\n","type":"template","slug":"agenda_home","options":{"admin_only":"","restrict_capability":"","capability_allowed":"","_dp_original":"328","_edit_lock":"1565803551:3","_edit_last":"3","pod_reference":"a:1:{s:3:\"pod\";s:7:\"concert\";}"}}}}
```

**Additional context**
Add any other context about the problem here.

**Possible Workaround**
Workaround for now is (obviously) not using the word 'status' for a label.

**Comment by jimtrue** [comment]
Can confirm this one. Add Relationship Field, Simple Custom Defined List, named 'Status'. Will act like it's saving but field will not exist upon saving.

**Comment by JoryHogeveen** [comment]
`status` is a reserved name, so that is correct.
The error handling should be fixed in #5453
